sqlx-core/Cargo.toml

```diff
@@ -49,7 +49,7 @@ ipnetwork = { workspace = true, optional = true }
 mac_address = { workspace = true, optional = true }
 uuid = { workspace = true, optional = true }
 
-async-io = { version = "1.9.0", optional = true }
+async-io = { version = "2.3.2", optional = true }
```

Comment:
This is actually undesirable because it means we'd no longer use the same I/O driver as `async-std`. It'd be on them to upgrade first.

Reply:
Fixed